app/src/store/cart.ts

```diff
@@ -95,6 +96,102 @@ export default function useCartStore() {
     }
   }
 
+  // convert the amount to a different token (direction allows for moving to/from the tokenAddress)
+  function getConvertedAmount(amount: number, tokenAddress: string, direction = 1) {
```

Comment:
Can you document with a comment whether `amount` is human-readable amount (e.g. 1 DAI) or raw/parsed number (e.g. 1e18 DAI). If the latter, make sure to change multiplication and division to use the `BigNumber` class

Reply:
These should always be human readable - I've added comments, thank you!